collect_app/src/main/java/org/odk/collect/android/activities/CaptureSelfieActivityNewApi.java

```diff
@@ -33,12 +33,20 @@
 
 import timber.log.Timber;
 
+import static org.odk.collect.android.utilities.PermissionUtils.checkIfCameraPermissionGranted;
+
 @TargetApi(Build.VERSION_CODES.LOLLIPOP)
 public class CaptureSelfieActivityNewApi extends CollectAbstractActivity {
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
+
+        if (!checkIfCameraPermissionGranted(this)) {
```

Comment:
I just tested this and when I revoke the camera permission and go back to this screen it just returns to the question. Is this behavior okay @lognaturel ? The user revoking the camera permission after granting it is highly unlikely so I think this is fine but a part of me would still show them a dialog explaining why I am closing the activity.